### ISSUE TITLE
Ender Tether appears to gradually prevent enderman spawning

### ISSUE THREAD

**Issue by Lilylicious** [Darkhax-Minecraft/Dark-Utilities#28]
Problem description: It appears that over time, the ender tether prevents endermen from spawning entirely.

Test setup: I placed down an Ender IO powered spawner set to endermen on a creative capacitor bank. I put in an octadic capacitor to speed up spawns. A draconic evolution mob grinder was used to kill endermen as they showed up. 

Test case 1: No tether, mobs spawned fine for 15 minutes. Test ended.

Test case 2: Tether placed. After a few spawn cycles, only one enderman spawned, on top of the tether (intended). After another few cycles, none spawned at all. Removing the tether does not appear to remove the issue.

Test case 3: Tested with a tether plus cursed earth, same issue. Endermen stopped spawning soon after placing the tether.

Issue appeared after updating from 1.1.5.96 to 1.1.6.99, using the Direwolf20 1.10.2 modpack. It strongly appears to be related to the ender tether spawn feature.

Edit: The issue disappears either over time or after server restart. Hard to tell which.

Guess as to cause: The Ender IO spawner checks for max nearby entities. Maybe there are entities being created but failing somehow, making the spawner think there's more around?

**Comment by clutchy42** [comment]
Would like to verify the same here using a non powered Enderman Spawner.

Process: On a Direwolf 20 1.10.2 1.1.2 server I used an Actually Additions Spawn Changer to transform a Blaze Mob Spawner into an Enderman Mob Spawner. Then using a the various movement and damage plates from Dark Utilities accompanied by an Ender Tether I created a farm in a 9x9x9 room. The process worked fine without issue. 

Issue: During the past week I updated my server to Direwolf 20 1.10.2 1.1.4. Along with it the latest stable build of Dark Utilities (1.1.6.99). I started a new world and I rebuilt the Enderman Farm to the exact specifications of the previous world. A few would spawn, seemingly break free of the tether and begin teleport all over the greater area the spawner was built in, and despite their observed models dying wherever they were no loot was dropping near the damage plates.

Action: Based on Lulylicious' similar observations I just rolled my servers Dark Utilities mod version back to 1.1.5.96 from 1.1.6.99. Restarted the server and without changing an item in the setup everything is working again.

It seems that the Ender Tether change made in 1.1.6.99 which states "Ender Tether now teleports enderman when they spawn as well" is somehow causing them to break in the process.

Before the rollback I was observing a repeated error in the logs:

17.12 12:36:13 [Server] Server thread/WARN Wrong location! (-35, -13) should be (-34, -13), EntityEnderman['Enderman'/330, l='Rebirth', x=-549.50, y=60.00, z=-204.50]


The specified "Wrong location!" was the spot where the Enderman Spawner exists in the World. 
There was also an additional output in the log that seemed to accompany the error above, but I cannot find it now that I am looking since I have wiped most older logs. Hope this helps though and thanks for your work!
